src/main/webapp/WEB-INF/web.xml

```diff
@@ -115,6 +125,16 @@
         <url-pattern>/*</url-pattern>
     </filter-mapping>
 
+    <filter-mapping>
```

Comment:
Is it possible to make these filters optional? That can only be turned on with a flag in the Studio configuration? Because it could be an issue, if the load balancer or reverse proxy doesn't overwrite the flags, or a client is using no reverse proxy.

Reply:
Both filters check the request and are skipped if the headers are not present, do you think we sill need a flag to disable them?